### PR TITLE
[TILE] Lightstep incident response TILE only integration with Datadog

### PR DESCRIPTION
**Pull request by deepakarumugham**
### What does this PR do?

Add [Lightstep incident response](https://lightstep.com/incident-response) tile in Datadog

### Motivation

To promote [Lightstep incident response's](https://lightstep.com/incident-response) integration with Datadog

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: lightstep_incident_response/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+[Lightstep Incident Response][1] seamlessly orchestrates alert triaging and on-call scheduling across Slack, Zoom, desktop, and mobile. Provides unified response context with integrated collaboration alongside on-call, alert, and incident management workflows.
+
+Lightstep Incident Response ingests data from Datadog through standard APIs and finds anomalous events before your business is impacted.
+
+## Setup
+
+### Installation
+
+To configure the integration and send data to Lightstep Incident Response, see the [Lightstep Incident Response-Datadog Integration][2].
```
Comment (alai97) [inline lightstep_incident_response/README.md:13]:
```suggestion
To configure the integration and send data to Lightstep Incident Response, see the [Lightstep Incident Response-Datadog Integration documentation][2]. 
```

File: lightstep_incident_response/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+To configure the integration and send data to Lightstep Incident Response, see the [Lightstep Incident Response-Datadog Integration][2]. 
+
+If you do not have a Lightstep Incident Response account, [sign up][3] to start your free trial.
+
+## Support
+
+Contact [Datadog support][4] or chat with a live agent after signing up in [Lightstep Incident Response][1].
+
+
```
Comment (alai97) [inline lightstep_incident_response/README.md:21]:
```suggestion
## Further Reading

Additional helpful documentation, links, and articles:

- [Datadog integration with Incident Response][2]

```

File: lightstep_incident_response/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Lightstep Incident Response
+
+## Overview
+
+[Lightstep Incident Response][1] seamlessly orchestrates alert triaging and on-call scheduling across Slack, Zoom, desktop, and mobile. Provides unified response context with integrated collaboration alongside on-call, alert, and incident management workflows.
```
Comment (alai97) [inline lightstep_incident_response/README.md:5]:
```suggestion
[Lightstep Incident Response][1] orchestrates alert triaging and on-call scheduling across Slack, Zoom, desktop, and mobile. Lightstep also provides unified response context with integrated collaboration along with on-call, alert, and incident management workflows.
```

**Review by alai97 — APPROVED** [review]
Looks good, copy nits!

**Comment by deepakarumugham** [comment]
@alai97 Can you please let me know, whom should I include to review and get this PR merged

**Comment by alai97** [comment]
Hey @deepakarumugham, thanks for accepting my doc suggestions! Someone from the Agent Integrations team will do a complete review of your PR first and then approve and merge 🥳

File: lightstep_incident_response/manifest.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b61db30e-3cbc-4dbe-b055-824f9e46e006",
+  "app_id": "lightstep--incident-response",
```
Comment (yzhan289) [inline lightstep_incident_response/manifest.json:4]:
```suggestion
  "app_id": "lightstep_incident_response",
```
Just to keep things consistent between directory name, metrics prefix, and app id

**Review by yzhan289 — CHANGES_REQUESTED** [review]

**Comment by Rohini-Ramakrishnappa** [comment]
@alai97 I have included the change suggested by you. There seems to be a permission issue due to which the "validations / build (pull_request)" is failing. Could you please help me merging this change?

File: lightstep_incident_response/manifest.json [hunk 5]
```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b61db30e-3cbc-4dbe-b055-824f9e46e006",
+  "app_id": "lightstep_incident_response",
```
Comment (yzhan289) [inline lightstep_incident_response/manifest.json:4]: Looks like our validator requires the `app_id` to not use `_`, could you change this to `lightstep-incident-response` instead?

Comment (yzhan289) [reply lightstep_incident_response/manifest.json:4]: That should fix the validations CI job.

Comment (Rohini-Ramakrishnappa) [reply lightstep_incident_response/manifest.json]: replaced the underscore with hyphen

**Comment by Rohini-Ramakrishnappa** [comment]
@yzhan289  fixed the validation issue, could you please help us get the changes merged?

**Comment by yzhan289** [comment]
> @yzhan289 fixed the validation issue, could you please help us get the changes merged?

Hi, sorry for the delay. I was OOO for a few days. I'll take a look at this now.

File: lightstep_incident_response/metadata.csv [hunk 6]
```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,description,orientation,integration,short_name
```
Comment (yzhan289) [inline lightstep_incident_response/metadata.csv:1]:
```suggestion
metric_name,metric_type,interval,unit_name,description,orientation,integration,short_name,curated_metric
```
We recently added this as a new column.

**Comment by yzhan289** [comment]
Hi @Rohini-Ramakrishnappa , I was syncing this tile on our staging, although it looks like you need a `CHANGELOG.md` file. The validations didn't catch this earlier, because I think this was recently added in our validations.

**Comment by Rohini-Ramakrishnappa** [comment]
> Hi @Rohini-Ramakrishnappa , I was syncing this tile on our staging, although it looks like you need a `CHANGELOG.md` file. The validations didn't catch this earlier, because I think this was recently added in our validations. You can look at #1275 for an example of the `CHANGELOG.md` requirement.

Hi @yzhan289 , added 'CHANGELOG.md' as suggested.

**Review by yzhan289 — APPROVED** [review]

**Comment by Rohini-Ramakrishnappa** [comment]
> 

@yzhan289  Would I be able to see the tile in the marketplace post-approval? are there any additional things to take care for me to get the publishing in place?

**Comment by yzhan289** [comment]
@Rohini-Ramakrishnappa you should be able to see the tile soon on app.datadoghq. I'm resolving a small issue on our end, but it should be resolved by tomorrow. Nothing else should be necessary to get it published!

**Pull request merged**